client/src/browse/BrowsePage.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+
+import Map from "./maps/Map.js";
+import ItemList from "./list/ItemList.js";
+import FilterBar from "./filterBar/FilterBar.js";
+
+const BrowsePage = (props) => {
+  const [lat, setLat] = useState(null);
+  const [lng, setLng] = useState(null);
+  const [itemData, setItemData] = useState([]);
+  const [filteredItems, setFilteredItems] = useState([]);
+  const [selectedItem, setSelectedItem] = useState(null);
+
+  useEffect(() => {
+    if (navigator.geolocation) {
+      navigator.geolocation.watchPosition(function (position) {
+        setLat(position.coords.latitude);
+        setLng(position.coords.longitude);
+      });
+    }
+  }, []);
+
+  useEffect(() => {
+    fetch("http://localhost:3000/local")
+      .then((res) => res.json())
+      .then((res) => {
+        const temp = [];
+        for (let i = 0; i < res.length; i += 100) {
+          temp.push(res[i]);
+        }
+        setItemData(temp);
+        setFilteredItems(temp);
+      });
+  }, []);
+
+  return (
+    <div>
+      <FilterBar
+        itemData={itemData}
+        setSelectedItem={setSelectedItem}
+        setFilteredItems={setFilteredItems}
+      />
+      <Container>
+        <Map
```

Comment:
@MarkAlperin just from personal experience a bit more specific names for styled components can help in readability later on. Especially if they are offloaded to a different file. So I would personally name "<Container>" -> "<MapContainer>" just to make it a bit more explicit. Not a request or requirement, just sharing my experience.

Reply:
replaced with a more descriptive name for styled-component